docs/data-sources/resource.md

```diff
@@ -29,7 +29,7 @@ data "twingate_resource" "foo" {
 
 - `address` (String) The Resource's address, which may be an IP address, CIDR range, or DNS address
 - `name` (String) The name of the Resource
-- `protocols` (Block List) By default (when this argument is not defined) no restriction is applied, and all protocols and ports are allowed. (see [below for nested schema](#nestedblock--protocols))
+- `protocols` (Block, Read-only) By default (when this argument is not defined) no restriction is applied, and all protocols and ports are allowed. (see [below for nested schema](#nestedblock--protocols))
```

Comment:
they're all read-only so why change only here?

Reply:
in the schema we have now separately list of attributes and list of blocks
and for the block we can't set Required/Optional/Computed, this available only for attribute fields

this additional `Read-only` add by doc generator, I'm not setting it 

```
resp.Schema = schema.Schema{
		Description: "Resources in Twingate represent any network destination address that you wish to provide private access to for users authorized via the Twingate Client application. Resources can be defined by either IP or DNS address, and all private DNS addresses will be automatically resolved with no client configuration changes. For more information, see the Twingate [documentation](https://docs.twingate.com/docs/resources-and-access-nodes).",
		Attributes: map[string]schema.Attribute{
			attr.ID: schema.StringAttribute{
				Required:    true,
				Description: "The ID of the Resource. The ID for the Resource can be obtained from the Admin API or the URL string in the Admin Console.",
			},
			// computed
			attr.Name: schema.StringAttribute{
				Computed:    true,
				Description: "The name of the Resource",
			},
			attr.Address: schema.StringAttribute{
				Computed:    true,
				Description: "The Resource's address, which may be an IP address, CIDR range, or DNS address",
			},
			attr.RemoteNetworkID: schema.StringAttribute{
				Computed:    true,
				Description: "The Remote Network ID that the Resource is associated with. Resources may only be associated with a single Remote Network.",
			},
		},
		Blocks: map[string]schema.Block{
			attr.Protocols: schema.SingleNestedBlock{
				Description: "By default (when this argument is not defined) no restriction is applied, and all protocols and ports are allowed.",
				Attributes: map[string]schema.Attribute{
					attr.AllowIcmp: schema.BoolAttribute{
						Computed:    true,
						Description: "Whether to allow ICMP (ping) traffic",
					},
				},
				Blocks: map[string]schema.Block{
					attr.TCP: &protocolSchema,
					attr.UDP: &protocolSchema,
				},
			},
		},
	}
```